src/app/course/event/display-question/display-question.component.ts

```diff
@@ -0,0 +1,10 @@
+import {Component} from '@angular/core'
+
+@Component({
+    selector: 'app-display-question',
+    templateUrl: './display-question.component.html',
+    styleUrls: ['./display-question.component.scss']
+})
+export class DisplayQuestionComponent {
```

Comment:
I don't think `DisplayQuestion` is a good component name. Maybe `CourseQuestionView`? or if you can come up with something better.

Reply:
Well this is only for displaying event questions